src/transformations/extent.jl

```diff
@@ -6,19 +6,24 @@ calculating and adding an `Extents.Extent` to all objects.
 
 This can improve performance when extents need to be checked multiple times.
```

Comment:
I think this could use a bit more comments. I wasn't familiar with this concept and it took a good few minutes to understand what was going on. Maybe a brief description of what extents are and how to access them after running.

Reply:
Ahh right, I'm assuming a lot of knowledge here. Definately more docs could help!

The ideas is you can use the extent as e.g. a fast `pointinpolygon` proxy. If its in the extent you need to actually check the polygon, if its not you know in a few ns.